9.secretstore/README.md

```diff
@@ -0,0 +1,144 @@
+
+Once you have a cluster, follow the steps below to deploy Dapr to it. For more details, look [here](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#installing-dapr-on-a-kubernetes-cluster)
+
+> Please note, that using the CLI does not support non-default namespaces.  
+> If you need a non-default namespace, Helm has to be used (see below).
+
+```
+$ dapr init --kubernetes
+ℹ️  Note: this installation is recommended for testing purposes. For production environments, please use Helm
+
+⌛  Making the jump to hyperspace...
+✅  Deploying the Dapr Operator to your cluster...
+✅  Success! Dapr has been installed. To verify, run 'kubectl get pods -w' in your terminal
+```
+
+## Step 2 - Configure a Secret in the Secret Store
+
+Dapr can use a number of different secret stores (AWS Secret Manager, Azure Key Vault, GCP Secret Manager, Kubernetes, etc) to retrieve secrets. For this demo, we'll use Kubernetes. For other secret stores, you can follow the instructions at https://github.com/dapr/docs/tree/master/howto/setup-secret-store
+
+1. Add your secret to a file ./secret
+2. Create the secret in the Kubernetes secret store. Note, the name of our secret here is "mysecret" and will be used in a later step.
+    ```
+    kubectl create secret generic mysecret --from-file ./secret
+    ```
+3. You can check that the secret is indeed stored in the Kubernetes secret store by running the command:
+    ```
+    kubectl get secret mysecret -o yaml
+    ```
+   You can see the output as below where the secret is stored in the secret store in Base64 encoded format
+   ```
+    % k get secret mysecret -o yaml
+        apiVersion: v1
+        data:
+        secret: eHl6OTg3Ngo=
+        kind: Secret
+        metadata:
+        creationTimestamp: "2020-05-20T20:20:09Z"
+        name: mysecret
+        namespace: default
+        resourceVersion: "2031800"
+        selfLink: /api/v1/namespaces/default/secrets/mysecret
+        uid: 64c60c3e-dce4-4c02-b5c1-8d4dbb7cbb8f
+        type: Opaque
+    ```
+
+
+## Step 3 - Deploy the Node.js App with the Dapr Sidecar
+
+```
+kubectl apply -f ./deploy/node.yaml
+```
+
+This will deploy our Node.js app to Kubernetes.
+
+You'll also see the container image that we're deploying. If you want to update the code and deploy a new image, see **Next Steps** section. 
+
+This deployment provisions an External IP.
+Wait until the IP is visible: (may take a few minutes)
+
+```
+kubectl get svc nodeapp
+```
+
+> Note: Minikube users cannot see the external IP. Instead, you can use `minikube service [service_name]` to access loadbalancer without external IP.
+
+Once you have an external IP, save it.
+You can also export it to a variable:
+
+```
+export NODE_APP=$(kubectl get svc nodeapp --output 'jsonpath={.status.loadBalancer.ingress[0].ip}')
```

Comment:
Let's add the equivalent command for windows as well

Reply:
fixed